tests/tools/discover_data/test_tool.py

```
@@ -0,0 +1,646 @@
+
+    # Create a mock Langfuse client
+    mock_langfuse = MagicMock()
+    monkeypatch.setattr(util.langfuse, "get_langfuse", lambda: mock_langfuse)
+
+    temporal = TemporalConstraint()
+    spatial = SpatialConstraint()
+
+    monkeypatch.setattr(
+        tool,
+        "extract_constraints",
+        lambda q, explicit_temporal, explicit_spatial: (temporal, spatial),
```

Ruff (0.14.14)

Unused lambda argument: `q`

(ARG005)